ts/Stock/StockToolsBindings.ts

```diff
@@ -2079,6 +2079,79 @@ const stockToolsBindings: Record<string, Highcharts.NavigationBindingsOptionsObj
             (annotation.options.events.click as any).call(annotation, {});
         }
     },
+    /**
+     * The Fibonacci Time Zones annotation bindings. Includes `start` and one
+     * event in `steps` array.
+     *
+     * @type    {Highcharts.NavigationBindingsOptionsObject}
+     * @product highstock
+     * @default {"className": "highcharts-fibonacci-time-zones", "start": function() {}, "steps": [function() {}], "annotationsOptions": {}}
+     */
+    fibonacciTimeZones: {
+        /** @ignore-option */
+        className: 'highcharts-fibonacci-time-zones',
+        // eslint-disable-next-line valid-jsdoc
+        /** @ignore-option */
+        start: function (
+            this: NavigationBindings,
+            e: PointerEvent
+        ): Annotation|void {
+            const coords = this.chart.pointer.getCoordinates(e),
+                coordsX = this.utils.getAssignedAxis(coords.xAxis),
+                coordsY = this.utils.getAssignedAxis(coords.yAxis);
+
+            // Exit if clicked out of axes area
+            if (!coordsX || !coordsY) {
+                return;
+            }
+
+            const navigation = this.chart.options.navigation,
+                options = merge(
+                    {
+                        type: 'fibonacciTimeZones',
+                        langKey: 'fibonacciTimeZones',
+                        typeOptions: {
+                            xAxis: coordsX.axis.options.index,
+                            yAxis: coordsY.axis.options.index,
+                            points: [{
+                                x: coordsX.value
+                            }]
+                        }
+                    },
+                    navigation.annotationsOptions,
+                    (navigation.bindings as any).fibonacciTimeZones.annotationsOptions
+                );
+
+            return this.chart.addAnnotation(options);
+        },
+        /** @ignore-option */
+        // eslint-disable-next-line valid-jsdoc
+        steps: [
+            function (
+                this: NavigationBindings,
+                e: PointerEvent,
+                annotation: Annotation
+            ): void {
+                const mockPointOpts = annotation.options.typeOptions.points,
+                    x = mockPointOpts && mockPointOpts[0].x,
+                    coords = this.chart.pointer.getCoordinates(e),
+                    coordsX = this.utils.getAssignedAxis(coords.xAxis),
+                    coordsY = this.utils.getAssignedAxis(coords.yAxis);
+
+                annotation.update({
+                    typeOptions: {
+                        xAxis: coordsX.axis.options.index,
+                        yAxis: coordsY.axis.options.index,
+                        points: [{
+                            x: x
+                        }, {
+                            x: coordsX.value
+                        }]
+                    }
+                });
```

Comment:
From what I see, you are only translating point. Don't you wanna use the `annotation.translatePoint()` method?

Reply:
No, sorry. I'm not translating the point, I'm updating the distance between 2 lines and then translating all the lines here https://github.com/highcharts/highcharts/blob/25c57a09fe2b0474ef6dc0c69a60bbb364fe991a/ts/Extensions/Annotations/Types/FibonacciTimeZones.ts#L269

Reply:
If I used `translatePoint()` here, I'd probably move (translate) only the first line.